biit_server/account_handler.py

```diff
@@ -73,12 +102,22 @@ def account_put(request):
     # return account.update(args)
 
     # TODO remove once db is implemented
-    return http200("Account Updated")
+    return jsonHttp200("Account Updated", auth)
 
 
 def account_delete(request):
-    """
-    Handles the account DELETE endpoint
+    """Handles the account DELETE endpoint
+    Validates data sent in a request then calls the database to remove the associated account
+
+    Args:
+        request: A request object that contains args with keys: email, token
+
+    Returns:
+        (json): Http 200 string response with refresh token and new token
```

Comment:
no token should be returned here since the account is deleted. So no token should be needed